specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -135,6 +139,15 @@ namespace Azure.Messaging.EventGrid {
     succeededLockTokens: string[];
   }
 
+  @doc("The result of the Reject operation.")
+  model RejectResult {
+    @doc("Array of LockToken values for failed cloud events.")
+    failedLockTokens: FailedLockToken[];
+
+    @doc("Array of LockToken values for succeeded cloud events.")
+    succeededLockTokens: string[];
```

Comment:
Why does `failedLockTokens` have a type of `FailedLockToken` but `succeededLockTokens` have type `string`? Either one of these is typed incorrectly or one of them is not an array of "LockTokens" as the name suggests.

Reply:
FailedLockToken contains additional information - the errorCode and errorDescription. So the docs need updating to clarify that it is an array of FailedLockToken values.

Reply:
Updating the docs is certainly welcome, but I think some users will still find the naming confusing.

A "LockToken" is a string. A "succeededLockToken" is a LockToken (string) that succeeded. A "failedLockToken" is _not_ a LockToken that failed -- it is a completely different type.

Rather than "failedLockToken", I think a better name would be "LockTokenFailure" -- since it describes the failure for a lock token.  Is it too late to make a name change?

Reply:
It's not really a LockTokenFailure - it's a failure in one of the settlement operations (acknowledge or release), and the lock tokens that can be used to correlate the failures with the events that were received via the receive operations are included in the payload.